pkgs/tools/text/grin/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, pythonPackages }:
+
+pythonPackages.buildPythonPackage rec {
+  name = "grin-1.2.1";
+  namePrefix = "";
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/g/grin/${name}.tar.gz";
+    sha256 = "1swzwb17wibam8jszdv98h557hlx44pg6psv6rjz7i33qlxk0fdz";
+  };
+
+  buildInputs = with pythonPackages; [ nose ];
+  propagatedBuildInputs = with pythonPackages; [ argparse ];
+
+  meta = {
+    homepage = https://pypi.python.org/pypi/grin;
+    description = "A grep program configured the way I like it.";
```

Comment:
I guess I am the maintainer, as the author won't be using nixos. I guess I add myself to `lib/maintainers.nix`.